lib/fs/fakefs.go

```diff
@@ -220,7 +231,6 @@ func (fs *fakefs) create(name string) (*fakeEntry, error) {
 		entry.mode = 0666
 		return entry, nil
 	}
-
```

Comment:
Seems weird to remove this empty line while adding two further down around `if fs.insens`. There's a few more emptly lines added/removed which look potentially spurious. I usually just go with what the existing code looks like (in here it seems to be a bit inconsistent, with some places with quite a few empty lines and others almost none - I tend to add them if I feel there's a sensible break (and I'd consider a returned branch as such)).

Reply:
Oh, regarding whitelines I just tend to do what `wsl` linter tells me to while leaving the existing code be, so feel free to nitpick. If you feel like there are ways to impove code readability, I should most definitely listen and do what's right.